src/c++/perf_analyzer/model_parser.cc

```diff
@@ -305,16 +288,36 @@ ModelParser::InitTorchServe(
 }
 
 cb::Error
-ModelParser::GetEnsembleSchedulerType(
-    const rapidjson::Document& config, const std::string& model_version,
-    std::unique_ptr<cb::ClientBackend>& backend, bool* is_sequential)
+ModelParser::DetermineComposingModelMap(
+    const std::vector<cb::ModelIdentifier>& bls_composing_models,
+    const rapidjson::Document& config,
+    std::unique_ptr<cb::ClientBackend>& backend)
+{
+  RETURN_IF_ERROR(AddBLSComposingModels(bls_composing_models, config));
+  RETURN_IF_ERROR(AddEnsembleComposingModels(config, backend));
+
+  return cb::Error::Success;
+}
+
+cb::Error
+ModelParser::AddBLSComposingModels(
+    const std::vector<cb::ModelIdentifier>& bls_composing_models,
+    const rapidjson::Document& config)
 {
-  const auto& sequence_itr = config.FindMember("sequence_batching");
-  if (sequence_itr != config.MemberEnd()) {
-    *is_sequential = true;
+  for (auto model : bls_composing_models) {
+    (*composing_models_map_)[config["name"].GetString()].insert(model);
```

Comment:
Is there any way to fetch the version? The output looks weird without the version. It is `version: `.

Reply:
The existing ensemble support did not do this either. I have updated it so that the version that is printed, even when it is not specified.

Reply:
https://github.com/triton-inference-server/client/pull/279/commits/c90725e32fac457729910217ce513337d736468d